### ISSUE TITLE
Luminosity ratio of selected item with its background is less than 3:1

### ISSUE THREAD

**Issue by Ram-G** [Azure/LogicAppsUX#3797]
### Describe the issue

1. Open Designer standalone (nx serve).
2. Add an action that uses folder picker editor.
3. Tab navigate to folders in the folder picker pop-out.
4. Observe that there is only a faint highlight around currently selected item.

**Expected:**
There should be a high contrast highlight or there should be a bounding box around the currently selected item.

**Actual:**
Currently selected item is only faintly highlighted.

### Are you using new designer or old designer

New Designer (With action panels)

### Screenshots or Videos

Not sufficient highlight around currently selected item:
![image](https://github.com/Azure/LogicAppsUX/assets/10837129/7bbb8fec-8d68-46dd-b8a7-ec3b7064c4dd)

Bounding box around button to navigate to next folder level:
![image](https://github.com/Azure/LogicAppsUX/assets/10837129/e2255c61-959d-4018-a877-63408614b066)

### Browser

Chrome

### Additional context

Related ADO bug: https://msazure.visualstudio.com/One/_workitems/edit/14566641
